backend/src-common/pom.xml

```diff
@@ -36,4 +36,23 @@
             </plugin>
         </plugins>
     </build>
+    <dependencies>
```

Comment:
there seem to be another (and maybe more) backend project(s) which is importing the same - cans this then be outdated?

Reply:
Yes, but that would make it an implicit dependency and `mvn dependency:analyze` would add a warning for that.